vshard/storage/init.lua

```diff
@@ -975,6 +982,10 @@ local function on_master_enable(new_func, old_func)
     end
 end
 
+local function on_bucket_event(new_func, old_func)
+    M._on_bucket_event(new_func, old_func)
```

Comment:
You should to `return M._on...()`. Without `return` you won't return the resulting installed function, which is quite useful when people set closures as triggers. Lets use that in the tests too.

Reply:
Moreover, the user may want to check the list of installed triggers with `vshard.storage.on_bucket_event()`. Let's also return values for the `on_master_enable/disable` too.

Reply:
Strange question. Why aren't `on_master_enable/disable` documented?  There's not even a ticket on doc repo about them.

Reply:
Lol. Actually, the trigger, created from lua doesn't return function (e.g. `net.box.on_schema_reload`). Trigger related functions, exported from C, do that (e.g.`box.sessions.on_auth`)